### ISSUE TITLE
An option to use the hotkeys with a controller 

### ISSUE THREAD

**Issue by runsrr** [ptyfyre/mandarine-neo#65]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

I want to be able to swap screens and close the game with a controller (for windows) 

### Why would this feature be useful?

It would be easier than pressing one of the keys on a keyboard every time you want to swap the screens or close the game.

**Comment by salamandercc37** [comment]
yes it would be great to be able to have the screen change directly on the controller and not on the computer keyboard

**Comment by runsrr** [comment]
The new update does not seem to have this function
